src/components/TableComponent.vue

```diff
@@ -83,19 +103,43 @@ onMounted(() => gettrack());
                     :key="i"
                     class="bg-white border-b dark:bg-gray-800 dark:border-gray-700"
                 >
-                    <th class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white" scope="row">
+                    <th
+                        v-if="roles?.includes('FEEDBACKGEVER')"
+                        class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white"
+                        scope="row"
+                    >
+                        {{ track.title }}
+                    </th>
+                    <td v-if="roles?.includes('FEEDBACKGEVER')" class="px-6 py-4">@{{ track.author.username }}</td>
+                    <th
+                        v-if="roles?.includes('MUZIEKPRODUCER')"
+                        class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white"
+                        scope="row"
+                    >
                         {{ track.id }}
                     </th>
-                    <td class="px-6 py-4">
+                    <td v-if="roles?.includes('MUZIEKPRODUCER')" class="px-6 py-4">
                         {{ track.title }}
                     </td>
                     <td class="px-6 py-4">
                         {{ track.genre }}
                     </td>
                     <td class="px-6 py-4">
-                        <span v-if='track.status[0] == "READY_TO_REVIEW"' class="bg-gray-100 text-gray-800 text-xs font-medium mr-2 px-2.5 py-0.5 rounded-full dark:bg-gray-900 dark:text-gray-300">Ready to review</span>
-                        <span v-if='track.status[0] == "PENDING_REVIEW"' class="bg-blue-100 text-blue-800 text-xs font-medium mr-2 px-2.5 py-0.5 rounded-full dark:bg-blue-900 dark:text-blue-300">Pending Review</span>
-                        <span v-if='track.status[0] == "REVIEWED"' class="bg-green-100 text-green-800 text-xs font-medium mr-2 px-2.5 py-0.5 rounded-full dark:bg-green-900 dark:text-green-300">Reviewed</span>
+                        <span
```

Comment:
Lijkt me redelijk onnodig om 3x hetzelfde neer te zetten als maar 1 stukje veranderd, dit kan je beter in de script regelen

Reply:
Nee